src/mudlet.cpp

```diff
@@ -2147,11 +2148,15 @@ void mudlet::deleteLine(Host* pHost, const QString& name)
     }
 }
 
-void mudlet::insertText(Host* pHost, const QString& name, const QString& text)
+bool mudlet::insertText(Host* pHost, const QString& windowName, const QString& text)
 {
     QMap<QString, TConsole*>& dockWindowConsoleMap = mHostConsoleMap[pHost];
-    if (dockWindowConsoleMap.contains(name)) {
-        dockWindowConsoleMap[name]->insertText(text);
+    const auto window = dockWindowConsoleMap.constFind(windowName);
```

Comment:
Why the change in accessing the relevant member of `dockWindowConsoleMap`? If we need to change to a `const` reference to the relevant `TConsole` instance it is sufficient to change:
```
    dockWindowConsoleMap[name]->insertText(text);
```
to
```
    dockWindowConsoleMap.value(name)->insertText(text);
```
and then return `true` or `false` as needed to indicate that the specified `TConsole` was found... :confounded: